site/src/Map.jsx

```diff
@@ -156,86 +159,99 @@ export default function Map({
     setZoom(event.target.getZoom());
   };
 
-  return (
```

Comment:
It's a bit hard to tell but I think the remainder of the changes here in the Map.jsx are just whitespace changes from the merge, right? i.e. no layer changes or anything have happened as a result of this PR?

Reply:
Yes, this should be resolved now and easier to review